integration/gethnetwork/geth_network.go

```diff
@@ -276,7 +276,7 @@ func (network *GethNetwork) StopNodes() {
 	for idx, process := range network.nodesProcs {
 		if process != nil {
 			wg.Add(1)
-			go func(process *os.Process) {
+			go func(process *os.Process, nodeNumber int) {
```

Comment:
Fixed a bug here, where the node number would be updated in the for loop, so it would look like the final node stopped a bunch of times.